src/cascadia/PublicTerminalCore/Source.cpp

```diff
@@ -0,0 +1,291 @@
+#include "pch.h"
+#include "HwndTerminal.hpp"

```

Comment:
Why is the header in `HwndTerminal.hpp` and the object module in `Source.cpp`? Shouldn't the names align?